### PR TITLE
Refactoring-and-Documentation

### PR DESCRIPTION
**Pull request by Gandalfhfd**
There have been plenty of changes, so it's time

### CODE REVIEW

File: POM/todo-angular-ts.ts [hunk 1]
```diff
@@ -147,25 +140,21 @@ export class AngularHomepage {
...
-        } catch (error) {
-            return false;
-        }
+    async locateTodoBySubstring(text: string): Promise<Locator> {
+        return this.page.getByRole('listitem').filter({ hasText: text });
     }
 
     /**
-     * Check that a todo with strictly the specified text (i.e. no whitespace) is present on the page.
+     * Check that a todo with the specified test, which has neither leading nor
```
Comment (KnightSky4) [inline POM/todo-angular-ts.ts:152]: Should say text not test
```suggestion
     * Check that a todo with the specified text, which has neither leading nor
```

File: POM/todo-angular-ts.ts [hunk 2]
```diff
@@ -229,23 +218,16 @@ export class AngularHomepage {
...
-     * @param text The unique text with which to locate a todo.
-     */
-    async deleteTodoByText(text: string): Promise<void> {
-        let targetTodo: Locator = this.page.getByRole('listitem').filter({ hasText: text });
-        await targetTodo.hover();
-        await targetTodo.getByRole('button', { name: '×' }).click();
-    }
-
     /**
      * Delete all todos matching text in an array.
```
Comment (KnightSky4) [inline POM/todo-angular-ts.ts:222]:
```suggestion
     * Delete all todos matching text.
```

File: POM/todo-angular-ts.ts (hunk 1 above)
Comment (Gandalfhfd) [reply POM/todo-angular-ts.ts:152]: good spot

**Review by KnightSky4 — CHANGES_REQUESTED** [review]

**Review by KnightSky4 — APPROVED** [review]
Thanks!

**Pull request merged**